App.py

```diff
@@ -0,0 +1,13 @@
+import dash
+from create_layout import create_layout
+from visualziation_callbacks import Visualization_callback
```

Comment:
Everything looks good! My only comment is that these 2 files hosting singular functions don't necessarily need to be split up since they're only used in this App.py file